02-peer-protocol.md

```diff
@@ -938,6 +960,38 @@ previously sent.
 A receiving node MAY ignore spurious message retransmission, or MAY
 fail the channel if they occur.
 
+### Rationale
+
+The effect of requirements above are that the opening phase is almost
+atomic: if it doesn't complete, it starts again.  The only exception
+is where the `funding_signed` message is sent and not received: in
+this case, the funder will forget the channel and presumably open
+a new one on reconnect; the other node will eventually forget the
+original channel due to never receiving `funding_locked`.
```

Comment:
"due to never receiving `funding_locked`" -> "due to never detecting the funding tx"?

The fundee doesn't really care about receiving a funding_locked, if it sees the funding tx on the blockchain (which maybe pays him `pushMsat`).